components/Dialogs/tests/unit/MDCAlertControllerRippleTests.m

```diff
@@ -0,0 +1,80 @@
+// Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <XCTest/XCTest.h>
+
+#import "MaterialDialogs.h"
+
+#import "MDCAlertControllerView+Private.h"
+
+#pragma mark - Subclasses for testing
+
+@interface MDCAlertController (Testing)
+@property(nonatomic, nullable, weak) MDCAlertControllerView *alertView;
+@end
+
+#pragma mark - Tests
+
+@interface MDCAlertControllerRippleTests : XCTestCase
+@property(nonatomic, nullable) MDCAlertController *alert;
+@end
+
+@implementation MDCAlertControllerRippleTests
+
+- (void)setUp {
+  [super setUp];
+
+  self.alert = [MDCAlertController alertControllerWithTitle:@"title" message:@"message"];
+  [self.alert addAction:[MDCAlertAction actionWithTitle:@"action1" handler:nil]];
+  [self.alert addAction:[MDCAlertAction actionWithTitle:@"action2" handler:nil]];
+}
+
+- (void)tearDown {
+  self.alert = nil;
+
+  [super tearDown];
+}
+
+/**
+ Test to confirm behavior of initializing a @c MDCAlerController without any customization.
+ */
+- (void)testDefaultAlertControllerBehaviorWithInkEnabled {
```

Comment:
Similarly here as in https://github.com/material-components/material-components-ios/pull/7596/files#r293630410